src/toil/batchSystems/gridengine.py

```diff
@@ -222,7 +222,8 @@ def supportsHotDeployment(cls):
 
     def __init__(self, config, maxCores, maxMemory, maxDisk):
         super(GridengineBatchSystem, self).__init__(config, maxCores, maxMemory, maxDisk)
-        self.gridengineResultsFile = self._getResultsFileName(config.jobStore)
+        assert config.jobStore.startswith('file:')
```

Comment:
I would extract the expected prefix into a local variable and reuse in the line below.